src/routes/_error.svelte

```diff
@@ -17,10 +17,7 @@
   {:else}
     <title>Something went wrong...</title>
   {/if}
-  <link rel="stylesheet" href="/css/page-components/header.css">
-  <link rel="stylesheet" href="/css/page-components/404.css">
-  <link rel="stylesheet" href="/css/page-components/403.css">
-  <link rel="stylesheet" href="/css/page-components/footer.css">
+  <link rel="stylesheet" href="/css/bundles/-error.min.css" />
```

Comment:
What is this dash?

Reply:
Well, the page is called `_error`, and in the script I replace all `_` with `-`, figured this is not a problem, so I kept it